.github/workflows/ci.yml

```diff
@@ -15,8 +15,6 @@ jobs:
     strategy:
       matrix:
         rust:
-          - stable
```

Comment:
Any reason to remove this?

Reply:
Yes, I assumed the CI should test only the MSRV. This assumption may not be correct - should I revert this one?

Reply:
Yeah, I'd like CI to run on both MSRV and current stable :)